src/Tizen.Telephony/Interop/Interop.Call.cs

```diff
@@ -30,31 +30,31 @@ internal static partial class Interop
     internal static partial class Call
```

Comment:
those Interops are internal classes, any comments here are just for internal use, could have added just one comment per such file

Reply:
Modified.